src/core/server/saved_objects/import/extract_errors.ts

```diff
@@ -37,11 +41,31 @@ export function extractErrors(
         originalSavedObject &&
         originalSavedObject.attributes &&
         originalSavedObject.attributes.title;
+
       if (savedObject.error.statusCode === 409) {
+        // find the correct title of the saved object that conflicts, in order to use it in the message box
+        let realTitle = title;
+
+        if (savedObjectsClient) {
+          try {
+            if (sourceSpaceId !== undefined) {
+              const resp = await savedObjectsClient.get(savedObject.type, savedObject.id, {
```

Comment:
Performing a `savedObjectsClient.get` for every conflict causes a separate request to Elasticsearch. We should determine all of the saved objects which have conflicts, and execute a `savedObjectsClient.bulkGet`.

Reply:
Fixed with commit cfd882d2f211b7a1cc132c74fa45cc8f9f5e8858 .